BE/src/main/java/yanse/airbnb/domain/image/MainImageRepository.java

```diff
@@ -0,0 +1,10 @@
+package yanse.airbnb.domain.image;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import yanse.airbnb.type.ImageType;
+
+import java.util.List;
+
+public interface MainImageRepository extends JpaRepository<Image, Long> {
```

Comment:
여기도 메서드를 보니 이미지 타입별로 조회하는 것 같은데 아래로 가는게 맞지 않을까요? 별도로 다르게 네이밍 한 이유가 있을까요?

```suggestion
public interface ImageRepository extends JpaRepository<Image, Long> {
```

Reply:
네이밍 변경하도록 하겠습니다 :)